dev-resources/puppetlabs/services/certificate_authority/certificate_authority_int_test/ca_true_test/master/conf/ssl/ca/requests/test_cert_ca_true.pem

```diff
@@ -0,0 +1 @@
+
```

Comment:
No.  If we delete this, git will take out the directory.

Or maybe I should figure out how to create the requests directory from inside the test?